### PR TITLE
Rds/cml/10 8

### PR DESCRIPTION
**Pull request by claperle**

### CODE REVIEW

File: docs/section10/10-8.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Airside systems – Rule 10-8  
+**Schema Version:** 0.0.12  
+**Mandatory Rule:** False   
+**Rule ID:** 10-8  
+ 
+**Rule Description:** Where multiple HVAC zones or residential spaces are combined into a single thermal block the cooling efficiencies (for baseline HVAC System Types 3 and 4) shall be based on the  equipment capacity of the thermal block divided by the number of HVAC zones or residential spaces.
```
Comment (mkarpman) [inline docs/section10/10-8.md:6]: I think this check is basically to verify that model reflects design documents. So it should be handled the same way as 10-5. Let's discuss on the call.

Comment (claperle) [reply docs/section10/10-8.md:6]: This is about the baseline efficiency; I think this RDS should be absorbed by 10-7 as it is covered there.

File: docs/section10/10-8.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+# Airside systems – Rule 10-8
```
Comment (claperle) [inline docs/section10/10-8.md:1]: ARCHIVED because this is captured in 10-7

**Comment by weilixu** [comment]
Closed because:
ARCHIVED because this is captured in 10-7

**Pull request closed without merging**